docs/data/sdks/typescript-ampli.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Typescript Ampli SDK
+description: The Amplitude Typescript SDK Installation & Quick Start guide.
+icon: material/language-typescript
+---
+
+
+!!! note
+    This page covers Browser JavaScript and TypeScript runtimes. All (Itly) runtimes are deprecated.
+     If you are still using an (Itly) runtime, see the **[migration guide](#migrating-from-an-itly-runtime)** to ugrade to the newest runtime. Docs for the Itly version are available **[here](/data/deprecated-sdks/browser)**.
+
+Amplitude Data supports tracking analytics events from Node.js apps written in JavaScript (ES6 and above) and TypeScript (2.1 and above). The generated tracking library is packaged as a CJS module.
+
+The tracking library exposes a function for every event in your team’s tracking plan. The function’s arguments correspond to the event’s properties and are strongly typed to allow for
+ code completion and compile-time checks.
+
+???tip "Linting with Prettier"
+
+    To prevent linting errors for eslint and tslint, the SDK-generated files have the following to diasable the linters: 
+
+    `/* tslint:disable */`
+
+    `/* eslint-disable */`
+    
+
+    There's no corresponding “in-code” functionality with Prettier. Instead, add the generated `path/to/ampli` to your `.prettierignore` file. You can get the path with `ampli pull`. See the [Prettier documentation](https://prettier.io/docs/en/ignore.html) for more information. 
+
+## Installation
+
+These instructions are also available from the **Implementation** page of your Amplitude Data workspace.
+
+### Install the Ampli CLI
+
+If you haven't installed the Ampli CLI, [install it now](/data/using-the-ampli-cli).
+
+### Install dependencies
+
+If you haven't already, install the core Amplitude SDK dependencies.
+
+=== "npm"
+
+    ```bash
+    npm install @amplitude/analytics-browser
+    ```
+
+=== "Yarn"
+
+    ```bash
+    yarn add @amplitude/analytics-browser
+    ```
+
+!!!note
+    Note: when using Ampli in the browser, we recommend loading `@amplitude/analytics-browser` as a module rather than as a JS snippet.
+
+### Pull the SDK into your project
+
+At the project root, run `pull` command.
+
+```bash
+ampli pull
+```
+
+This prompts you to log in to your workspace and select a source.
+
+=== "TypeScript"
+
+    ```bash
+    ➜ ampli pull sourcename
+    Ampli project is not initialized. No existing `ampli.json` configuration found.
+    ? Create a new Ampli project here? Yes
+    Organization: Amplitude
+    Workspace: My Workspace
+    Source: sourcename
+    Runtime: Browser/TypeScript (2.0)
```

Comment:
remove the (2.0). It will be TypeScript after GA.

Reply:
+1